test/controllers/medias_controller_test.rb

```diff
@@ -591,7 +591,7 @@ def setup
     Media.any_instance.stubs(:get_canonical_url).returns(true)
     Media.any_instance.stubs(:try_https)
     WebMock.enable!
-    WebMock.disable_net_connect!
+    WebMock.disable_net_connect!(allow: [/minio/])
```

Comment:
@caiosba Why does just allowing minio fix this?

Reply:
Because this was the failure:

```
WebMock::NetConnectNotAllowedError: Real HTTP connections are disabled. Unregistered request: HEAD http://minio:9000/testbucket2/
```

We need to allow it so cache can be stored. It was flaky because maybe some other test was doing this or cache writing to the file system was not happening.